partition_snapshot_reader.hh

```diff
@@ -223,13 +215,9 @@ private:
 
     void on_new_range() {
         if (_current_ck_range == _ck_range_end) {
-            opt_reversed_range = std::nullopt;
             _end_of_stream = true;
             push_mutation_fragment(mutation_fragment_v2(*_schema, _permit, partition_end()));
         } else {
-            if constexpr (Reversing) {
-                opt_reversed_range = query::reverse(*_current_ck_range);
-            }
```

Comment:
Could we remove Reversing?

Reply:
I guess not, it still needs to know it's scanning the source backwards.

Reply:
You are right. We cannot.